roles/openshift_logging/tasks/install_elasticsearch.yaml

```diff
@@ -63,6 +63,7 @@
 - name: Generate PersistentVolumeClaims for Ops
   include: "{{ role_path}}/tasks/generate_pvcs.yaml"
   vars:
+    es_pvc_pool: []
```

Comment:
for the sake of uniformity and being explicit, can we add this also for the non-ops version of this task?